NSS-Evening-E22/pos-system-outlaw-bandanas
# Tjp revenue page functionality

## Description
Added functionality to the revenue page. When clicking to go to the revenue page, will be met with tip total, total revenue, and other totals from the closed orders

## How to Test
Go to Homepage
Click "View Revenue"
Revenue should render all data from closed orders
